config/config.go

```diff
@@ -18,26 +18,26 @@ const (
 	ENV_IDENTITY_CERT_FILE        = "IDENTITY_CERT_FILE"
 	ENV_IDENTITY_PRIVATE_KEY_FILE = "IDENTITY_PRIVATE_KEY_FILE"
 
-	ENV_FILESERVER_ADDRESS              = "FILESERVER_ADDRESS"
-	ENV_FILESERVER_PORT                 = "FILESERVER_PORT"
-	ENV_FILESERVER_CREDENTIALS_USERNAME = "FILESERVER_CREDENTIALS_USERNAME"
-	ENV_FILESERVER_CREDENTIALS_PASSWORD = "FILESERVER_CREDENTIALS_PASSWORD"
-	ENV_FILESERVER_ROOT_DIRECTORY       = "FILESERVER_ROOT_DIRECTORY"
+	ENV_SERVER_ADDRESS                       = "SERVER_ADDRESS"
+	ENV_SERVER_PORT                          = "SERVER_PORT"
+	ENV_SERVER_CREDENTIALS_USERNAME          = "SERVER_CREDENTIALS_USERNAME"
+	ENV_SERVER_CREDENTIALS_PASSWORD          = "SERVER_CREDENTIALS_PASSWORD"
+	ENV_SERVER_STATIC_CONTENT_ROOT_DIRECTORY = "SERVER_STATIC_CONTENT_ROOT_DIRECTORY"
 )
 
 // USED FOR YAML
-type FileServer struct {
-	Address        string               ",omitempty"
-	Port           int                  ",omitempty"
-	Credentials    security.Credentials ",omitempty"
-	Root_Directory string               ",omitempty"
+type Server struct {
+	Address                       string               ",omitempty"
```

Comment:
I guess this `",omitempty"` is for yaml.  It's a little bit different for json but I'll assume this is what you want.